### DISCUSSION TITLE
Disabling dynamic constraints/using HiGHS as B&B solver

### DISCUSSION THREAD

**Discussion by fuglede** [ERGO-Code/HiGHS#1503] [Q&A] [answered]
Is there a way to use the MIP solver as a pure B&B solver using LP relaxations? I'm happy if there's just an effective way, e.g. using various trivial values for things like `mip_pool_age_limit`, `mip_lp_age_limit`, `mip_pool_soft_limit` to avoid adding dynamic constraints, but I wasn't able to find one. I'm also happy with a pointer a la "you would have to comment out this piece of code, then rebuild" if it's not possible directly.

Main motivation being to get a numerical estimate of the number of nodes to visit for an otherwise theoretical study of the complexity of a particular class of MIPs.

**Comment by jajhall** [comment, marked as answer]
I don't think so. The MIP solver isn't designed to have its performance compromised to this extent, sorry.

**Reply by fuglede** [reply]
Noted, thanks for the answer!
